frontend/src/components/oligos/OligoList.js

```diff
@@ -69,6 +75,10 @@ function OligoList(props) {
       <br />
       { isAuthenticated ? (
         <React.Fragment>
+          <form>
+            <input type='text' className='input' placeholder='Search...' name='q' />
+          </form>
+          <ul> </ul>
```

Comment:
Is this line needed?

Reply:
It is not needed. It adds a small amount of space between the search bar and the table, which I thought looked nice. But it is purely cosmetic and can be removed. I've attached screenshots of the difference so you can let me know what you prefer.

With Space:
<img width="333" alt="space" src="https://user-images.githubusercontent.com/43936166/152920946-d9ff72f8-f0eb-42da-bae9-88672ff66002.png">

Without Space:
<img width="227" alt="no space" src="https://user-images.githubusercontent.com/43936166/152920983-97361816-4c87-48df-9046-b71d82d01d9f.png">